plaso/engine/profilers.py

```diff
@@ -62,6 +65,15 @@ def __init__(self, identifier, configuration):
     self._sample_file = None
     self._start_time = None
 
+  def _Write(self, content):
```

Comment:
Unclear from the method name what is written, maybe rename to `_WriteSample` or `_WriteString`

Reply:
Done